CoreScriptsRoot/CoreScripts/PurchasePromptScript2.lua

```diff
@@ -1549,3 +1763,61 @@ if IsNativePurchasing then
 		nativePurchaseFinished(wasPurchased)
 	end)
 end
+
+local function switchGui(activeGui, newVRMode)
```

Comment:
Hm, I suppose not, but it's nice that the implementation of the UI is opaque to the functional code. Maybe I'll go through and add some small helper functions to abstract that and get rid of the hotswapped table.